src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -96,6 +103,16 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /** Find and return the Conversation with the given ID. */
+  public Conversation getConvoWithID(UUID id) {
```

Comment:
nit: be consistent with either "conversation" or "convo". "conversation" seems to be the established convention in this file.